lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -85,6 +87,9 @@ sub _add_routes {
       if (@parameters) {
         $op_spec->{parameters} = [@parameters, @{$op_spec->{parameters} || []}];
       }
+      if ($op_spec->{security}) {
```

Comment:
You have inheritance for path parameters, using the same logic you can implement inheritance for the global "Security Requirement Object" (http://swagger.io/specification/#securityRequirementObject) ?

Not sure how important this is for us, since we have a separate scope for each operation object anyway, which overloads the global "Security Requirement Object".
To me, global inheritance doesn't look very difficult to add, so I would recommend adding it for the sake of specification completeness.